### ISSUE TITLE
how to do the augmentation for training?

### ISSUE THREAD

**Issue by seyeeet** [kazuto1011/deeplab-pytorch#103]
Thanks for our code, I have a clarification question and appreciate your answer.
Let say I am augmenting the image, lets say I resize the image and then pad it and then crop a fix size.
for the padding areas, should I assign class 0 (background) or class 255 (which is ignore class)?
in other words, should I backpropagate the loss for the pad areas and make my network to consider them as 0?

Thanks a lot for your help.

**Comment by kazuto1011** [comment]
A typical way is to pad with 0 and label with 255, and the 255 areas are ignored in computing loss. Whatever the network predicts on the padded areas in training, no problem if the network never sees the padding in testing.

**Comment by seyeeet** [comment]
thanks for your answer
